packages/examples/src/proxy-scripts/festo-live.ts

```diff
@@ -110,7 +110,7 @@ Promise.all(fetchArray).then(async (tdArray: ThingDescription[]) => {
     }, 5000);
 });
 
-// exposed Thing toward Oracle IoT Cloud Service
+// exposed Thing toward IoT Cloud Service
 WoT.produce({
```

Comment:
Will this script still work?

Reply:
@wiresio ?

Reply:
The mentioned files are related with `binding-oracle` and should be moved there. Maintenance then depends on the maintainer of the binding.